source/lambda/customResourceKendraDataSource/lambda_function.py

```diff
@@ -20,56 +20,58 @@ def on_create(event, context):
     )
     
     print("covid FAQ created")
-                                            
-    create_data_source_response = kendra_client.create_data_source(
-        Name='DUSCovidDataset',
-        IndexId= kendraIndexId,
-        Type='S3',
-        Configuration={
-            'S3Configuration': {
-                'BucketName': dataBucketName,
-                'InclusionPrefixes': [
-                    'documents/'
-                ]
+    
+    # create s3 folders
+    s3_client = boto3.client('s3', region_name=os.environ['AWS_REGION'])
+    s3_client.put_object(Bucket=os.environ['BULK_PROCESSING_BUCKET'], Key=('documentDrop' +'/'))
+    s3_client.put_object(Bucket=os.environ['BULK_PROCESSING_BUCKET'], Key=('kendraPolicyDrop' +'/'))
+    
+    # copy the covid pdfs into the bulk processing bucket for processing
+    # get a list of all covid pdfs and their associated kendra policy
+    response = s3_client.list_objects_v2(Bucket=dataBucketName, MaxKeys=1000)
+
+    s3_resource = boto3.resource('s3', region_name=os.environ['AWS_REGION'])
```

Comment:
Is there a specific reason to use resource? Could we reuse the s3_client defined above and the corresponding

s3_client.copy() API?

Reply:
yes, i could look into that